scipy/optimize/_constraints.py

```diff
@@ -249,6 +250,57 @@ def __init__(self, constraint, x0, sparse_jacobian=None,
         self.bounds = (lb, ub)
         self.keep_feasible = keep_feasible
 
+    def is_feasible(self, x):
```

Comment:
It doesn't sound familiar to me. Feasible point, feasible region, but feasible constraint --- I'm not sure if it is a correct term.

How about `is_satisfied`?